README.md

```diff
@@ -102,9 +102,18 @@ Nomad binary in the `bin` and `$GOPATH/bin` folders:
 
 ```sh
 $ make dev
-...
-$ bin/nomad
-...
+```
+
+Running `consul` is required before starting the nomad agent:
```

Comment:
Perhaps change this to:

> Optionally run Consul to enable service discovery and health checks:

Running Consul is not _required._ The Consul related features will not work if Consul is not running, but the rest of Nomad will function properly.

Reply:
I don't think we want to include that at all actually.  Would be okay with this PR if it just kept: 
And finally start the nomad agent:

```sh
 +$ sudo bin/nomad agent -dev
```

Reply:
@schmichael Fixed. :)

@dadgar When I was getting started with it I didn't know I had to run consul and the error messages was a bit confusing for me. I ended up spending some time on it before getting help on the project's gitter channel. With this change I hope it saves some time for  someone else new to the project.